src/current/cockroachcloud/alerts-page.md

```diff
@@ -29,16 +29,16 @@ The [Customer-Managed Encryption Keys (CMEK)]({% link cockroachcloud/cmek.md %})
 
 If you receive the alert repeatedly, verify the following:
 
-- The key is still enabled in their cloud provider KMS. 
-- The role or account used to access the key still has valid permissions. 
+- The key is still enabled in their cloud provider KMS.
```

Comment:
this whitespace commit suggests this is not a merge but a manual port. is that correct?

doesn't that mean when we merge `cloud-2.0` back into `main` we're going to potentially lose work that was done on `main` that didn't get manually ported over here?

Reply:
We will have conflicts to resolve, we won't lose work. I cherry-picked your PR's commit before you merged it, resolved Cloud-2.0-specific conflicts, then poked around looking for other fully-qualified links that were in `cloud-2.0` only. Sometimes I opened a file in my editor to make an edit, and then whitespace may have gotten trimmed. Some areas your PR changed had already been previously edited in `cloud-2.0` also, so the contents here are different from the ones in `main`.